src/pages/RecipeDetails/index.js

```diff
@@ -178,12 +179,12 @@ const mapDispatchToProps = (dispatch) => ({
 });
 
 RecipeDetails.propTypes = {
-  match: PropTypes.objectOf(PropTypes.string).isRequired,
+  match: PropTypes.objectOf(PropTypes.boll).isRequired,
   history: PropTypes.objectOf(PropTypes.string).isRequired,
   recipeFetching: PropTypes.bool.isRequired,
   recipeFetch: PropTypes.func.isRequired,
   recipe: PropTypes.objectOf(PropTypes.string).isRequired,
-  recommendationsFetch: PropTypes.func.isRequired,
+  recommendationsFetch: PropTypes.bool.isRequired,
```

Comment:
recomendations fetch é uma função
```suggestion
  recommendationsFetch: PropTypes.func.isRequired,
```